bitcash/network/rates.py

```diff
@@ -4,7 +4,7 @@
 from time import time
 
 import requests
-
+from bitcash.network import session
```

Comment:
This results in a circular import

Reply:
Moved session initialization to separate module, `coverage run -m pytest -m "not regtest" -rw` passes now.